### ISSUE TITLE
[Web] [Accessibility][11yMAS][Name, Role and Value] [MAS4.1.2] [Incorrect role is defined for “overview” control under "Filter by Product" Combo box in the Navigation List.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab#680]
**User Experience:**
User who rely on AT will not get proper information about “overview” control if role is not defined properly which provides the purpose of control.
 
**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0
Tools used: Developer tool
**Attachments:**
![mas4 1 2_role is incorrect](https://user-images.githubusercontent.com/33054846/32368149-ce5525d0-c0aa-11e7-8f4c-d9626c980a2a.png)
 
**Repro Steps:**
1. Open the URL: http://scriptlabdocsdemo.azurewebsites.net/Office%20Dev%20Center%20-%20Docs%20-%20Build%20your%20first%20Excel%20add-in.html .
2. Navigate to “overview” control and open developer tool.
3. Verify role is defined properly for “overview” control or not. 
 
**Current Behavior:** 
Incorrect role is defined for “Overview” control under "Filter by Product" Combo box in the Navigation List. Role is defined as Link for Overview.

**Expected Behavior:**
Role should be defined as “Menu” because Overview control is expanding after pressing enter and is having items inside it.

**Note:** 
1. Same Issue is reproducible for All controls under “overview” in the navigation list.
2.Same issue is repro for the [MAC + safari][ Build info: OS Build: Mac OS Sierra 0.12.6 Browser: Safari 11.0.1] Incorrect role is defined for the Overview control under the Filter by product combo box in the navigation list.
 
**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}
 
**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=4.1.2
